examples/electron-asar/package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "main": "./app.js",
   "dependencies": {
-    "ipfsd-ctl": "*"
+    "ipfsd-ctl": "file:../.."
```

Comment:
Ah, you were already doing it correctly here but then you required it again using a path